src/sdk/namespace_fs.js

```diff
@@ -329,7 +333,7 @@ class NamespaceFS {
              * @typedef {{
              *  key: string,
              *  common_prefix: boolean,
-             *  stat?: fs.Stats,
+             *  stat?: nb.NativeFSStats,
```

Comment:
this nb.NativeFSStats type needed in the LRU cache as well

Reply:
you're right! fixed.